cadence/transactions/kittyItems/add_nft_images_for_new_kind.cdc

```diff
@@ -0,0 +1,27 @@
+import KittyItems from "../../contracts/KittyItems.cdc"
+ 
+ 
```

Comment:
It would be good to add a high level description and some comments for this transaction